eo-maven-plugin/src/main/java/org/eolang/maven/TranspileMojo.java

```diff
@@ -158,16 +160,26 @@ public void exec() throws IOException {
                 this.targetDir.toPath().resolve(TranspileMojo.DIR),
                 TranspileMojo.EXT
             );
+            final String src = tojo.get(AssembleMojo.ATTR_EO);
             if (
                 target.toFile().exists()
                     && target.toFile().lastModified() >= file.toFile().lastModified()
+                    && target.toFile().lastModified() >= Paths.get(src).toFile().lastModified()
             ) {
                 Logger.info(
                     this, "XMIR %s (%s) were already transpiled to %s",
                     Save.rel(file), name, Save.rel(target)
                 );
             } else {
-                saved += this.transpile(input, target);
+                final List<Path> paths = this.transpile(src, input, target);
+                for (final Path path : paths) {
+                    this.transpiledTojos.value()
```

Comment:
@Karina5005 this seems a bit wrong, since you will have multiple lines in the file with the same "primary key". How about you use this structure of CSV:

```
Java | XMIR2
```

The "key" is the location of `.java` file (unique) and the next column is the location of `.xmir`.

Reply:
@yegor256 I don't understand what's wrong, ` transpile` returns a list of paths of generated java files. I use them as keys. Everything is done as you described, am I missing some detail?
Now I use structure of CSV:
`id | xmir2`
Where id is path to .java file